patches/openwrt/0900-replace-banner-with-freifunk.patch

```diff
@@ -15,12 +15,12 @@ index 799f7e5..628447c 100644
 - |   -   ||  _  |  -__|     ||  |  |  ||   _||   _|
 - |_______||   __|_____|__|__||________||__|  |____|
 -          |__| W I R E L E S S   F R E E D O M
-+  _____        _  __             _
-+ |  ___|      (_)/ _|           | |
-+ | |_ _ __ ___ _| |_ _   _ _ __ | | __
-+ |  _| '__/ _ \ |  _| | | | '_ \| |/ /
-+ | | | | |  __/ | | | |_| | | | |   <
-+ \_| |_|  \___|_|_|  \__,_|_| |_|_|\_\
++   _____ _     _    _  ____  _   _ 
++  / ____| |   | |  | |/ __ \| \ | |
++ | |  __| |   | |  | | |  | |  \| |
++ | | |_ | |   | |  | | |  | | . ` |
++ | |__| | |___| |__| | |__| | |\  |
++  \_____|______\____/ \____/|_| \_|                                                              
```

Comment:
@mweinelt 
Maybe like this? But there is no need to merge this pull request. ^^